### PR TITLE
e2e storage: update OWNERS

### PR DESCRIPTION
**Pull request by pohly**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:

This removes several peope who are no longer active with reviews and adds
myself as owner and reviewer.

#### Does this PR introduce a user-facing change?
```release-note
NONE
```

/assign @msau42

### CODE REVIEW

File: test/e2e/storage/OWNERS [hunk 1]
```diff
@@ -9,17 +9,21 @@ approvers:
...
-- jeffvance
-- copejon
 - verult
 - davidz627
 - xing-yang
+- pohly
 labels:
 - sig/storage
+
+emeritus_reviewers:
```
Comment (msau42) [inline test/e2e/storage/OWNERS:26]: Can you also move rootfs and davidz627 to emeritus as well?

And we can pull in the sig-storage-approvers/reviewers aliases

Comment (pohly) [reply test/e2e/storage/OWNERS:13]: Done. Let's also update that alias list.

**Comment by pohly** [comment]
/assign @wojtek-t

For approval of OWNERS change.

**Pull request merged**